### PR TITLE
chore: split babel config to standalone file and update vite config

### PR DESCRIPTION
**Pull request by daskyrk**
## What this PR does / why we need it:
* support babel plugin in vite for using jsx-control-statements
* move erda-icon used theme-colors into this file, for it's a CommonJs format file, can not import from es file in vite build.
* rename import from 'app/common' to 'common', make the bable import plugin work correctly.
* use a different app file as vite entry.
## Does this PR introduce a user interface change?

❎ No

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |              |
| 🇨🇳 中文    |              |

## Does this PR need be patched to older version?
❎ No

### CODE REVIEW

File: shell/app/layout/pages/page-container/page-container.tsx [hunk 1]
```diff
@@ -187,7 +187,7 @@ const PageContainer = ({ route }: IProps) => {
           >
             {MainContent}
           </div>
-          <MessageCenter show={showMessage} />
+          {/* <MessageCenter show={showMessage} /> */}
```
Comment (McDaddy) [inline shell/app/layout/pages/page-container/page-container.tsx:190]: Does this change related with vite?  or it's a requirement?

Comment (daskyrk) [reply shell/app/layout/pages/page-container/page-container.tsx]: Nope, comment for test, reverted.

File: shell/babel.config.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
+const overwriteMap = {
+  table: true,
+  select: true,
+  tag: true,
+  'range-picker': true,
+};
+
+const overwriteCssMap = {
+  table: 'antd/es/table/style',
+  select: 'antd/es/select/style',
```
Comment (McDaddy) [inline shell/babel.config.js:23]: `table` & `select` seems is using antd style and don't need ignore. Still required to have these two properties here?

Comment (daskyrk) [reply shell/babel.config.js]: Yes, it's useless, removed.

**Review by McDaddy — APPROVED** [review]

**Pull request merged**